flutter/test/sentry_flutter_test.dart

```diff
@@ -1,128 +1,143 @@
-import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:mockito/mockito.dart';
 import 'package:package_info_plus/package_info_plus.dart';
 import 'package:sentry/sentry.dart';
 import 'package:sentry_flutter/sentry_flutter.dart';
 import 'package:sentry/src/platform_checker.dart';
-
 import 'mocks.dart';
-import 'mocks.mocks.dart';
 import 'sentry_flutter_util.dart';
 
-void main() {
-  TestWidgetsFlutterBinding.ensureInitialized();
+/// These are the integrations which should be added on every platform.
+/// They don't depend on the underlying platform.
+final platformAgnosticIntegrations = [
```

Comment:
I added a lists for every platform and type of integration. Based on these lists we know which platform should have which integration. I think it quite easy to change if add support for more integrations (or add integrations on existing platforms).